python/lib/dcoscli/dcoscli/tables.py

```diff
@@ -422,8 +423,8 @@ def _str_to_datetime(datetime_str):
     """
     if not datetime_str:
         return None
-    datetime_str = datetime_str.split('+')[0]
-    return datetime.datetime.strptime(datetime_str, "%Y-%m-%dT%H:%M:%S.%f")
+    # Used to parse ISO 8601 formatted date strings.
+    return dateutil.parser.parse(datetime_str)
```

Comment:
Can it be done without an extra package? eg. https://docs.python.org/3.5/library/datetime.html#datetime.date.isoformat

Reply:
So what we need is a function that parses an ISO 8061 string and returns a `datetime` object. I found this https://docs.python.org/3.5/library/datetime.html#datetime.datetime.isoformat but it does exactly the opposite of what we need, turning a `datetime` object into a string. I have not found anything in pythons standard library that would be able to parse the variety of ISO 8061 formatted strings like `dateutil.parser.parse()`.